packages/itwin/ec3-widget/src/components/Templates.tsx

```diff
@@ -98,15 +98,15 @@ export const Templates = ({ onClickCreate, onClickTemplateTitle, onExportResult
         ) : templates.length === 0 ? (
           <>
             <EmptyMessage message={EC3Widget.translate("noTemplateMsg")} />
-            <Button startIcon={<SvgAdd />} onClick={onClickCreate} styleType="high-visibility" title="Add New Template">
-              Add New Template
+            <Button startIcon={<SvgAdd />} onClick={onClickCreate} styleType="high-visibility" title={EC3Widget.translate("addNewTemplate")}>
+              {EC3Widget.translate("addNewTemplate")}
             </Button>
           </>
         ) : (
           <div className="ec3w-templates-list">
             <div className="ec3w-toolbar" data-testid="ec3-templates">
               <div className="ec3w-toolbar-left">
-                <Button startIcon={<SvgAdd />} onClick={onClickCreate} styleType="high-visibility" title="New Template">
+                <Button startIcon={<SvgAdd />} onClick={onClickCreate} styleType="high-visibility" title={EC3Widget.translate("newTemplate")}>
                   New
```

Comment:
The "New" string should also be translated.

Reply:
"Export", "Delete", there is still some remaining.